odoo/modules/migration.py

```diff
@@ -226,18 +226,39 @@ def compare(version):
                     name, ext = os.path.splitext(os.path.basename(pyfile))
                     if ext.lower() != '.py':
                         continue
-                    mod = None
                     try:
                         mod = load_script(pyfile, name)
-                        _logger.info('module %(addon)s: Running migration %(version)s %(name)s' % dict(strfmt, name=mod.__name__))
-                        migrate = mod.migrate
-                    except ImportError:
-                        _logger.exception('module %(addon)s: Unable to load %(stage)s-migration file %(file)s' % dict(strfmt, file=pyfile))
-                        raise
-                    except AttributeError:
-                        _logger.error('module %(addon)s: Each %(stage)s-migration file must have a "migrate(cr, installed_version)" function' % strfmt)
-                    else:
-                        migrate(self.cr, installed_version)
-                    finally:
-                        if mod:
-                            del mod
+                    except ImportError as e:
+                        raise ImportError('module %(addon)s: Unable to load %(stage)s-migration file %(file)s' % dict(strfmt, file=pyfile)) from e
```

Comment:
The usage of `strfmt` pre-date python3 support. We can use f-strings now.
```suggestion
                        raise ImportError(f'module {addon}: Unable to load {stage}-migration file {pyfile}') from e
```

Reply:
It's a false problem as the current code, even without using f-strings, has still the same issue that this check warn about.